rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json

```diff
@@ -289,19 +289,19 @@
         ],
         "options": [
           {
-            "label": "French",
```

Comment:
As this seems to be used at several places. We can use a $ref with a common definition.

Reply:
Because in this case we only have 4 languages